grakn-core/src/main/java/ai/grakn/util/GraqlSyntax.java

```diff
@@ -200,6 +201,18 @@ public String toString() {
             return ImmutableMap.copyOf(accepted);
         }
 
+        private static Map<Method, Map<Algorithm, Map<Parameter, Object>>> argumentsDefault() {
+            Map<Method, Map<Algorithm, Map<Parameter, Object>>> defaults = new HashMap<>();
```

Comment:
if we are making the value maps immutable maybe let's make this map immutable as well?

Reply:
The map is returned via `ImmutableMap.copyOf(...)` now.